js/WebAPI/DocumentEditor.md

```diff
@@ -8,7 +8,7 @@ keywords: documenteditor,syncfusion,documenteditor webapi
 api: /api/js/ejdocumenteditor
 ---
 
-## Import
```

Comment:
Check meta details and h1 tag will be above 20 to below 70 characters.

Reply:
Done